modules/fts/pages/fts-troubleshooting.adoc

```diff
@@ -109,6 +109,18 @@ For If searching for the following is slow: lowFreqTerm highFreqTerm, try +lowFr
 +
 Although it is a different search, it should perform better and may be acceptable.
 
+*Q: Why are the queries getting rejected with http status code 429?*
+
+At a higher level, this shows that the cluster is under provisioned in terms of the ftsMemoryQuota. Whenever FTS service receives a query, it tries to forecast the memory requirements for performing that search request. And the query is rejected when fts service realises that the cumulative memory usage of the service goes beyond the set memory quota limit by executing the incoming query. 
+The memory alloted to fts service could be used by any of the following concurrent operations.
+
+* Parallel queries handled by the system.
+* Heavy concurrent indexing of data.
+* Parallel rebalance operations.
+* Complex queries with higher from/size parameters. 
+
+Users could fix this by following the right sizing guidelines/bumping the ftsMemoryQuota or by adding more fts nodes.
```

Comment:
Recommend using second person as follows:
"You can fix this by following the right sizing guidelines, increasing the ftsMemoryQuota, or by adding more Search nodes."

Reply:
done @amarantha-k , thanks!